test/testworker.js

```diff
@@ -231,7 +231,9 @@ export default class TestWorker extends EventEmitter {
     return await Promise.all(graph.tasks.map(async (task) => {
       // Note: that we assume runId 0 here which is fine locally since we know
       // the number of runs but not safe is we wanted to test reruns.
-      return await this.fetchTaskStats(task.taskId, 0);
+      var LocalLiveLog = require('../lib/features/local_live_log');
```

Comment:
Do we need to import this here or just import this globally in testworker.js instead of two different locations where it's imported?  Also, what do you think about moving that method out of the class and just have local_live_log.js export that function to be imported independently?  Do you see a strong case for this to be a class method?  I'm impartial either way, just curious.